### DISCUSSION TITLE
Convert Local Image to Upload scalar type

### DISCUSSION THREAD

**Discussion by DayaJuang** [strawberry-graphql/strawberry#3216] [Q&A] [answered]
I am now working on uploading file using strawberry.file_uploads Upload, I want to ask is there any way to convert my file/image in my local directory to Upload type for testing purpose ? My code will be like this:

image_path = "path/to/your/image.jpg"

with open(image_path, "rb") as file:
// Read the content of the file
    file_content = file.read()

// I want to convert the image here and process it

**Comment by DoctorJohn** [comment, marked as answer]
Hi, I would recommend using a Python GraphQL client for testing. They take care of the GraphQL multipart file upload protocol which can be tricky to handle by hand.
